cypress/commons/constants/generic/TestConstants.js

```diff
@@ -45,7 +45,11 @@ export const API_ENDPOINT = {
   DATASET_TWINGRAPH: URL_ROOT + '/.*/datasets/((d|D)-[\\w]+)/twingraph',
   RUNNERS: URL_ROOT + '/.*/runners',
   RUNNER: URL_ROOT + '/.*/runners/((r|R)-[\\w]+)',
-  RUNNER_DEFAULT_SECURITY: URL_ROOT + '/.*/runners/((r|R)-[\\w]+)/security/default',
+  START_RUNNER: URL_ROOT + '/.*/runners/((r|R)-[\\w]+)/start', // Endpoint to start the runner
+  RUNNER_STATE: URL_ROOT + '/.*/runs/(run-[\\w]+)/status',
+  SIMULATION_RUNNER_DEFAULT_SECURITY: URL_ROOT + '/.*/runners/((r|R)-[\\w]+)/security/default',
```

Comment:
`SIMULATION_RUNNER_DEFAULT_SECURITY` could be renamed to `RUNNER_DEFAULT_SECURITY`, because the endpoint is not specific to scenarios (what we call "simulation runners").
Same thing for `API_REGEX.SIMULATION_RUNNER_DEFAULT_SECURITY` :slightly_smiling_face:

Reply:
I created a new constant because the regex stored in `RUNNER_DEFAULT_SECURITY` doesn't match needed endpoint:
`'/.*/runners/((**d|D**)-[\\w]+)/security/default'`
It matches dataset security for some reasons, so I created a new one :slightly_smiling_face: